docs/UserGuide.adoc

```diff
@@ -171,6 +171,7 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+=======
```

Comment:
I will let this change pass. Will do another PR to patch formatting in UserGuide.